apps/bfd-model/bfd-model-rif/src/main/resources/db/migration/V27__Create_BeneficiaryMonthlytable.sql

```diff
@@ -0,0 +1,142 @@
+    "medicareStatusAugCode", "entitlementBuyInAugInd", "hmoIndicatorAugInd",
+    "partCContractNumberAugId", "partCPbpNumberAugId", "partCPlanTypeAugCode",
+    "partDContractNumberAugId", "partDPbpNumberAugId", "partDSegmentNumberAugId",
+    "partDRetireeDrugSubsidyAugInd", "medicaidDualEligibilityAugCode", "partDLowIncomeCostShareGroupAugCode"
+    FROM "Beneficiaries"
+    WHERE "beneEnrollmentReferenceYear" is not null;
+
+INSERT INTO "BeneficiaryMonthly" 
+    SELECT  "beneficiaryId", TO_DATE(CONCAT("beneEnrollmentReferenceYear", '/09/01'), 'YYYY/MM/DD') as "yearMonth", "fipsStateCntySeptCode",
+    "medicareStatusSeptCode", "entitlementBuyInSeptInd", "hmoIndicatorSeptInd",
+    "partCContractNumberSeptId", "partCPbpNumberSeptId", "partCPlanTypeSeptCode",
+    "partDContractNumberSeptId", "partDPbpNumberSeptId", "partDSegmentNumberSeptId",
+    "partDRetireeDrugSubsidySeptInd", "medicaidDualEligibilitySeptCode", "partDLowIncomeCostShareGroupSeptCode"
+    FROM "Beneficiaries"
+    WHERE "beneEnrollmentReferenceYear" is not null;
+
+INSERT INTO "BeneficiaryMonthly" 
+   SELECT  "beneficiaryId", TO_DATE(CONCAT("beneEnrollmentReferenceYear", '/10/01'), 'YYYY/MM/DD') as "yearMonth", "fipsStateCntyOctCode",
+    "medicareStatusOctCode", "entitlementBuyInOctInd", "hmoIndicatorOctInd",
+    "partCContractNumberOctId", "partCPbpNumberOctId", "partCPlanTypeOctCode",
+    "partDContractNumberOctId", "partDPbpNumberOctId", "partDSegmentNumberOctId",
+    "partDRetireeDrugSubsidyOctInd", "medicaidDualEligibilityOctCode", "partDLowIncomeCostShareGroupOctCode"
+    FROM "Beneficiaries"
+    WHERE "beneEnrollmentReferenceYear" is not null;  
+
+INSERT INTO "BeneficiaryMonthly" 
+    SELECT  "beneficiaryId", TO_DATE(CONCAT("beneEnrollmentReferenceYear", '/11/01'), 'YYYY/MM/DD') as "yearMonth", "fipsStateCntyNovCode",
+    "medicareStatusNovCode", "entitlementBuyInNovInd", "hmoIndicatorNovInd",
+    "partCContractNumberNovId", "partCPbpNumberNovId", "partCPlanTypeNovCode",
+    "partDContractNumberNovId", "partDPbpNumberNovId", "partDSegmentNumberNovId",
+    "partDRetireeDrugSubsidyNovInd", "medicaidDualEligibilityNovCode", "partDLowIncomeCostShareGroupNovCode"
+    FROM "Beneficiaries"
+    WHERE "beneEnrollmentReferenceYear" is not null;  
+
+INSERT INTO "BeneficiaryMonthly" 
+    SELECT  "beneficiaryId", TO_DATE(CONCAT("beneEnrollmentReferenceYear", '/12/01'), 'YYYY/MM/DD') as "yearMonth", "fipsStateCntyDecCode",
+    "medicareStatusDecCode", "entitlementBuyInDecInd", "hmoIndicatorDecInd",
+    "partCContractNumberDecId", "partCPbpNumberDecId", "partCPlanTypeDecCode",
+    "partDContractNumberDecId", "partDPbpNumberDecId", "partDSegmentNumberDecId",
+    "partDRetireeDrugSubsidyDecInd", "medicaidDualEligibilityDecCode", "partDLowIncomeCostShareGroupDecCode"
+    FROM "Beneficiaries"
+    WHERE "beneEnrollmentReferenceYear" is not null;
```

Comment:
Did we ever get a final answer as to why we have all the null `beneEnrollmentReferenceYear` records? Regardless, I'd suggest adding a comment here explaining why we're leaving them out -- even if it's just "we don't know why there are null values for this column, but we'll figure that out later in BFD-12345."

Reply:
Thinking about it a bit more, @jzulim, with all the records you found that had a null `beneEnrollmentReferenceYear`, did any of them have non-null values for the _other_ columns here?

Reply:
Added comment